### PR TITLE
v648-2 edits

### LINKED ISSUE #694: 2019 V648-2 Changes
Update edit V648-2 for 2019 collection to include 3 as an acceptable response.

If Sex of Co-Applicant or Co-Borrower equals 6, then Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname must equal 2 or 3.

### PR DESCRIPTION
**Pull request by MichaelFurey17**
Close #694 

from [Platform Issue 2774](https://github.com/cfpb/hmda-platform/issues/2774)

### CODE REVIEW

File: 2019/python/lar_constraints.py [hunk 1]
```diff
@@ -485,11 +485,11 @@ def v648_const(self, row):
 		"""1) If Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname equals 2,
 			then Sex of Co-Applicant or Co-Borrower must equal 1, 2, 3 or 6.
 		2) If Sex of Co-Applicant or Co-Borrower equals 6, then Sex of Co-Applicant or Co-Borrower Collected on the
-			Basis of Visual Observation or Surname must equal 2."""
+			Basis of Visual Observation or Surname must equal 2 or 3."""
 		if row["co_app_sex_basis"] == "2" and row["co_app_sex"] not in ("1", "2", "3", "6"):
 			row["co_app_sex"] = random.choice(("1", "2", "3", "6"))
 		if row["co_app_sex"] == "6":
```
Comment (Kibrael) [inline 2019/python/lar_constraints.py]: Please make a comment that code 3 is drawn from platform changes.

https://github.com/cfpb/hmda-platform/issues/2774

File: 2019/python/rules_engine.py [hunk 2]
```diff
@@ -1125,10 +1125,10 @@ def v648_1(self):
 	def v648_2(self):
 		"""An invalid Sex data field was reported.
 		2) If Sex of Co-Applicant or Co-Borrower equals 6, then
-		Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname must equal 2."""
+		Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname must equal 2 or 3."""
 		field = "Co-Applicant Sex Basis"
```
Comment (Kibrael) [inline 2019/python/rules_engine.py]: Please add a comment that the inclusion of 3 is from: https://github.com/cfpb/hmda-platform/issues/2774

File: 2019/python/lar_constraints.py (hunk 1 above)
Comment (MichaelFurey17) [reply 2019/python/lar_constraints.py]: Change has been implemented in recent commit.

File: 2019/python/rules_engine.py (hunk 2 above)
Comment (MichaelFurey17) [reply 2019/python/rules_engine.py]: change has been implemented in recent commit.

**Pull request merged**